packages/http-client-csharp/generator/TestProjects/CadlRanch/http/authentication/api-key/tspCodeModel.json

```diff
@@ -51,6 +51,7 @@
        "$id": "7",
        "Name": "endpoint",
        "NameInRequest": "endpoint",
+       "Description": "Service host",
```

Comment:
This change is due to the TCGC API migration. Previously we use the following logic:
```typescript
Description: parameter.__raw ? getDoc(sdkContext.program, parameter.__raw) : undefined
```
And now it's changed to:
```typescript
Description: parameter.doc
```

But in TCGC, the endpoint parameter is created without a raw type: https://github.com/Azure/typespec-azure/blob/dedf76c89a00f206fb6e79051e2dfff0b7e7d95d/packages/typespec-client-generator-core/src/package.ts#L507

So previously all `Description` properties are actually `undefined`. Now they have a value.